Source/Core/AudioCommon/CubebUtils.cpp

```diff
@@ -16,6 +16,38 @@
 
 static ptrdiff_t s_path_cutoff_point = 0;
 
+ScopedThreadAccess::ScopedThreadAccess()
+{
+#ifdef _WIN32
+  // COINIT_MULTITHREADED seems to be the better choice here given we handle
+  // multithread access safely ourself. It should also offer better performance.
+  // Note that calls to this from the main thread will return RPC_E_CHANGED_MODE
+  // as some external library is calling CoInitialize with COINIT_APARTMENTTHREADED
+  // without uninitializing it. We keep going.
+  // cubeb specifies we should use COINIT_MULTITHREADED:
+  // https://github.com/kinetiknz/cubeb/issues/416, but that can't be a problem
+  result = CoInitializeEx(NULL, COINIT_MULTITHREADED);
```

Comment:
```suggestion
  result = CoInitializeEx(nullptr, COINIT_MULTITHREADED);
```

Reply:
Again, copied from cubeb, NULL is fine, unless there is a rule to not use it in Dolphin, not really worth changing.

Reply:
There is a rule to not use it in Dolphin.